src/borg/helpers.py

```diff
@@ -269,6 +269,17 @@ def get_keys_dir():
     return keys_dir
 
 
+def get_key_nonces_dir():
+    """Determine where to store the local nonce high watermark"""
+
+    xdg_config = os.environ.get('XDG_CONFIG_HOME', os.path.join(get_home_dir(), '.config'))
+    nonces_dir = os.environ.get('BORG_KEY_NONCES_DIR', os.path.join(xdg_config, 'borg', 'key-nonces'))
```

Comment:
Sure. It was meant as the nonce being a mutable property of the key and thus stored at distinct location but still logically related to the key.
If you prefer BORG_NONCES_DIR i’ll use that.